crates/rolldown/src/bundler.rs

```diff
@@ -26,6 +30,9 @@ pub struct Bundler {
   pub(crate) resolver: SharedResolver,
   pub(crate) file_emitter: SharedFileEmitter,
   pub(crate) _log_guard: Option<FlushGuard>,
+  pub(crate) last_module_table: ModuleTable,
```

Comment:
I prefer `previous_module_table`, what do you think

Reply:
Done